targets/python37-jupyter-pytorch-tensorflow-jupyterlab/Dockerfile

```diff
@@ -1,14 +1,14 @@
 # ==================================================================
 # module list
 # ------------------------------------------------------------------
-# python        3.6       (apt)
+# python        3.7       (apt)
 # jupyter       latest    (pip)
 # jupyterlab    latest    (pip)
-# pytorch       1.2.0     (pip)
+# pytorch       1.5.1     (pip)
 # tensorflow    2.0.0-rc1 (pip)
 # ==================================================================
 
-FROM nvidia/cuda:10.0-cudnn7-devel-ubuntu18.04
+FROM pytorch/pytorch:1.5.1-cuda10.1-cudnn7-devel
```

Comment:
Big change :wink:

Reply:
Wait, I overlooked this. What about Tensorflow? Will it work on this image as it was?

Reply:
I don't see any reason why TF would not work. Drivers the same, just version of Cuda toolkit has been changed.

Reply:
Indeed, itself it is build from [`nvidia/cuda:10.1-cudnn7-devel-ubuntu16.04`](https://github.com/pytorch/pytorch/blob/fe415589a964697a78186189bd4815541f853b47/docker/pytorch/Dockerfile#L1)